core/src/mindustry/ui/fragments/MenuFragment.java

```diff
@@ -28,6 +28,30 @@ public class MenuFragment{
     private Button currentMenu;
     private MenuRenderer renderer;
     private Seq<MenuButton> customButtons = new Seq<>();
+    public MenuButton[] desktopButtons;
```

Comment:
Should this be `final`?

Reply:
let me remind you that final variables cannot be changed, even in classes that contain it

Reply:
...the line thats highlighted by this comment?

Reply:
don't make it final